imagetest/test_suites/network/setup.go

```diff
@@ -74,6 +78,16 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 	}
 	vm2.UseGVNIC()
 	vm1.RunTests("TestPingVMToVM|TestDHCP|TestDefaultMTU")
-	vm2.RunTests("TestAlias|TestGVNIC")
+	vm2.RunTests("TestAlias")
+	if strings.Contains(t.Image, "debian-10") {
+		// GVNIC Guest OS Feature not set on Debian 10
+		return nil
+	}
+	vm3, err := t.CreateTestVM("vm3")
```

Comment:
why are we adding a third VM? it doesn't seem related to the debian-10 change

Reply:
oh but why not do something like this:

```go
if strings.Contains(t.Image, "debian-10") {
  vm2.RunTests("TestAlias")
} else {
  vm2.RunTests("TestAlias|TestGVNIC")
}
```

Reply:
Oh yes that seems better. Thanks